### ISSUE TITLE
Group comparison: Enable mutation and copy number tabs when we have more than two groups

### ISSUE THREAD

**Issue by alisman** [cBioPortal/cbioportal#5990]
@zhx828 can you describe the proposed solution here?

**Comment by schultzn** [comment]
This will be for the next release. But we should minimally enable bar plots that show frequency differences between groups for each gene. Something like the below.

![image](https://user-images.githubusercontent.com/11892704/56810042-59a9c680-6803-11e9-9bc1-cc15231a180e.png)

Users should be able to define how many genes they want to show and be able to choose a sort order (e.g., decreasing order by max frequency in any cancer type, decreasing order of average frequency). 

Statistics would be nice to have but could become tricky if there are too many groups... but I am sure we can come up with something.

**Comment by jjgao** [comment]
We should have the bar plot when there are two groups as well. Maybe it's also useful when there is only one group? @schultzn

**Comment by jjgao** [comment]
@kalletlak maybe you backend code for calculating p-values for the clinical data tab can be used here?

**Comment by jjgao** [comment]
@alisman I noticed the milestone is 3.0.0. Are we plan to include this in the first release?

**Comment by schultzn** [comment]
Maybe it’s worth postponing the release for this? We waited this long for this feature, so we can wait a little longer. I think it would be nice to come out with a (mostly) fully functional feature.

**Comment by kalletlak** [comment]
@schultzn @jjgao could you come up with some mock UI design on where to show the plot and all filtering options

**Comment by jjgao** [comment]
@kalletlak you can add the chart on top as the third chart once #6030 is done. Maybe you can take #6030 as well?

![image](https://user-images.githubusercontent.com/11892704/56975605-29359580-6b3f-11e9-9eaa-eaac71d1d906.png)

**Comment by jjgao** [comment]
For sorting, let's give three options:

1) decreasing order by max frequency in any cancer type (default)
2) decreasing order of average frequency
3) decreasing order of p value

Can we add a star when q is significant (<=0.05)?

**Comment by kalletlak** [comment]
@jjgao do we want the table if there are more than 2 groups?

**Comment by jjgao** [comment]
@kalletlak I think so. I can be similar to `Clinical` tab. I guess `Chi-square Test` will be used?

If we do that, `Mutations` and `CNAs` tab will have different tests based on the number of groups. Should we do the same for `Clinical` tab?

**Comment by jjgao** [comment]
For two groups:

- keep the current table
- Change "Enriched in group x" into a dropdown
- add bar plot to the top

For three groups or more:
- add bar plot to the top

In the table:
- Gene
- Cytoband
- Freq per group
- Static score (chi-square score?)
- p
- q
- if possible, `Enriched in`

For filter:
- Significant only
- Change "Enriched in group x" into a dropdown
